Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModal.swift

```diff
@@ -21,6 +21,7 @@ import SwiftUI
 public struct BPKAppSearchModal: View {
     let title: String
     @Binding var inputText: String
+    let inputPrefix: BPKSearchInputSummary.InputPrefix
```

Comment:
should this not be optional?

Reply:
Alright, they should all have a prefix 👍

Reply:
It seems, we it has not to be nil, otherwise it would be a regular BPKTextField according to requirements